commands/curation/curationaudit_test.go

```diff
@@ -626,31 +667,28 @@ func getTestCasesForDoCurationAudit() []testCase {
 		{
 			name:          "maven tree - one blocked package",
 			tech:          techutils.Maven,
-			pathToPreTest: filepath.Join(TestDataDir, "projects", "package-managers", "maven", "maven-curation", "pretest"),
+			pathToProject: filepath.Join("projects", "package-managers", "maven", "maven-curation"),
+			pathToTest:    "test",
+			pathToPreTest: "pretest",
 			preTestExec:   "mvn",
 			funcToGetGoals: func(t *testing.T) []string {
-				rootDir, err := os.Getwd()
-				assert.NoError(t, err)
-				// set the cache to test project dir, in order to fill its cache with dependencies
-				callbackPreTest := clienttestutils.ChangeDirWithCallback(t, rootDir, filepath.Join("..", "test"))
+				// We want to fill the cache with dependencies before the tests to only resolve in the test the blocked package
```

Comment:
```suggestion
				// We want to fill the cache with dependencies before the tests start, so we resolve during the test only the blocked package
```
This is hard to understand